src/mixins/amalgamation-mixin.ts

```diff
@@ -278,9 +276,4 @@ export default class AmalgamationMixin extends Vue {
       (business.type === AmlTypes.LEAR && business.legalType === CorpTypeCd.BC_ULC_COMPANY)
     )
   }
-
-  /** True if there is a BC company in the table. */
-  get isAnyBcCompany (): boolean {
```

Comment:
I believe this is not needed. This is since that's being taken care of the other rule `needBcCompany`.

Reply:
OK, so needBcCompany() check if all are foreign. What is the value of isAllForeign if there are no companies at all? (Ie, what does "every" do?) Maybe it doesn't matter because we don't evaluate any rules if there are no tings, but it's still possibly wrong.

I think I'd prefer using `!isAnyBcCompany` for the rules instead. It's probably slower (more complex) but I think it's also more accurate. Thoughts?

Reply:
I agree Sev! I've deleted the isAllForeign getter and instead, I'm using now the isAnyBcCompany getter.

Reply:
LGTM!

Reply:
(Feel free to resolve this conversation.)